ui/src/message/MessagesStore.ts

```diff
@@ -100,6 +100,13 @@ export class MessagesStore {
         this.createEmptyStatesForApps(this.appStore.getItems());
     };
 
+    @action
+    public refreshByApp = async (appId: number) => {
+        const state = this.stateOf(appId);
+        const result = await this.fetchMessages(appId, 0).then((resp) => resp.data);
+        state.messages.replace(result.messages);
```

Comment:
I think we should refresh all messages because when refreshing only one application the "All Messages"-Tab will not get updated resulting in inconsistent state.
So we can use `this.clearAll()` to discard our state and `this.loadMore()` to get the most recent messages.